src/Root.tsx

```diff
@@ -31,80 +33,7 @@ const NotebookApp: React.FC = () => {
   // rather than dynamic sync payload updates, as LiveStore doesn't support
   // runtime sync payload changes
 
-  // Periodic auth validation to detect token expiry
-  useEffect(() => {
-    const validateAuth = async () => {
-      const isValid = await isAuthStateValid();
-      if (!isValid) {
-        console.warn("Auth state is invalid, forcing reload with reset");
-        const url = new URL(window.location.href);
-        url.searchParams.set("reset", "auth-invalid");
-        window.location.href = url.toString();
-      }
-    };
-
-    // Check auth state every 30 seconds for faster detection
-    const interval = setInterval(validateAuth, 30 * 1000);
-
-    // Also check immediately
-    validateAuth();
-
-    return () => clearInterval(interval);
-  }, []);
-
-  // Listen for WebSocket connection errors that might indicate auth issues
-  useEffect(() => {
-    let reconnectAttempts = 0;
-    const maxReconnectAttempts = 3;
-
-    const validateAuth = async () => {
-      const isValid = await isAuthStateValid();
-      if (!isValid) {
-        console.warn("Auth state is invalid, forcing reload with reset");
-        const url = new URL(window.location.href);
-        url.searchParams.set("reset", "auth-invalid");
-        window.location.href = url.toString();
-      }
-    };
-
-    const handleVisibilityChange = () => {
-      if (document.visibilityState === "visible") {
-        // When user returns to tab, validate auth state
-        isAuthStateValid().then((isValid) => {
-          if (!isValid) {
-            console.warn("Auth state invalid on tab focus, reloading");
-            const url = new URL(window.location.href);
-            url.searchParams.set("reset", "auth-focus-check");
-            window.location.href = url.toString();
-          }
-        });
-      }
-    };
-
-    // Monitor for repeated WebSocket failures that might indicate auth issues
-    const handleBeforeUnload = () => {
-      reconnectAttempts++;
-      if (reconnectAttempts >= maxReconnectAttempts) {
-        console.warn("Multiple reconnection attempts, checking auth state");
```

Comment:
I just nuked all this as it seemed a bit excessive. Feel free to bring back some version of it if it makes sense